config/game_options.txt

```diff
@@ -390,11 +390,11 @@ SILICON_MAX_LAW_AMOUNT 12
 ## You probably want humans on your space station, but technically speaking you can turn them off without any ill effect
 ROUNDSTART_RACES human
 
-## Races that are strictly worse than humans that could probably be turned on without balance concerns
+## Races that are strictly worse than humans that could probably be turned on without balance concerns, but will probably be off due to the strict NO FUN policy
```

Comment:
remove this line, also we have a fucking config on the server. we don't use this config.

Reply:
Do you want me to close this?

Because this is how you get me to close this.

Reply:
it's true though

Reply:
this is why you keep getting permabanned everywhere morty